content/about/notes/0.8.md

```diff
@@ -58,5 +50,5 @@ and about [`istioctl gen-deploy`](/docs/reference/commands/istioctl/#istioctl%20
 
 - A gateway with virtual services pointing to a headless service won't work ([Issue #5005](https://github.com/istio/istio/issues/5005)).
 
-- There are [problems with Kubernetes 1.10.2](https://github.com/istio/istio/issues/5723).
+- There is a [problem with Google Kubernetes Engine 1.10.2](https://github.com/istio/istio/issues/5723).
```

Comment:
Is this only a prob with GKE?

Reply:
yes, its a problem with the Container Optimizes OS on GKE only

Reply:
ok great, thanks for clarifying it!